.circleci/config.yml

```diff
@@ -78,7 +119,6 @@ commands:
             echo "ENABLE_ABOUT_PAGE=$ENABLE_ABOUT_PAGE" >> .env
             echo "SLACK_URL=$SLACK_URL" >> .env
             echo "SLACK_CHANNEL_NAME=$SLACK_CHANNEL_NAME" >> .env
-            cd ..
```

Comment:
Are you sure that removing this doesn't break anything?

Reply:
yeah, each 'run' section does not relate to the next, it opens unique shell

Reply:
I mean why did you remove the `cd ..` in a lot of the steps?

Reply:
ah got it I understand